docs/userGuide/syntax/attributes.md

```diff
@@ -62,5 +62,11 @@ don't add a space for **inline**{.text-danger} markdown
 
 <!-- Reader facing features -->
 <div id="examples" class="d-none">
-<include src="attributes.md#list-example" />
+
```

Comment:
minor nit would be that you can remove
id="list-example" since it is no longer referenced rite.

Same for the one below

Reply:
Thanks for pointing this out! Fixed in 2b876aa